tests/testthat/test-int-latent_individual.R

```diff
@@ -68,6 +68,74 @@ test_that("epidist.epidist_latent_individual Stan code has no syntax errors and
   expect_no_error(mod_gamma$compile())
 })
 
+test_that("epidist.epidist_latent_individual fits and the MCMC converges  in the gamma delay case", { # nolint: line_length_linter.
+  skip_on_cran()
+  set.seed(1)
+  
+  gamma_family <- epidist_family(
+    prep_obs_gamma,
+    family = "gamma",
+    dpars = c("mu", "shape"),
+    links = c("log", "log"),
+    lb = c(0, 0),
+    ub = c(NA, NA)
+  )
+
+  gamma_formula <- epidist_formula(prep_obs_gamma)
+  gamma_formula$pforms$sigma <- NULL
+  gamma_formula <- brms::bf(gamma_formula, as.formula("shape ~ 0"))
+  
+  fit_gamma <- epidist(
+    data = prep_obs_gamma,
+    family = gamma_family,
+    prior = brms::prior("normal(0, 3)", class = "Intercept"),
+    formula = gamma_formula,
```

Comment:
Having to change the names here makes this not a great experience I think for the user. If we don't want to unpick this suggests moving more things internal to epidist (and passing a character string for family with all current arguments done internally. People would then be able to make code level changes via a new class

Reply:
At the moment I've mostly been trying to get the model to work in the `test-int-latent_individual` script i.e. this code is not a suggestion of the best way to present to the user.

But yes I think these issues do suggest more thought or moving more things internal is required.

Reply:
See comment about starting with a base brms fit